Makefile

```diff
@@ -378,9 +379,8 @@ $(BUILD_DIR)/gene-profiles.ttl: $(BUILD_DIR)/monarch-data.ttl $(SPARQL)/geneProf
 
 # Generate absences.ttl
 $(BUILD_DIR)/absences.ttl: $(BUILD_DIR)/phenoscape-data-kb.ofn $(SPARQL)/absences.sparql
-	$(ROBOT) query \
-    	-i $< \
-    	--query $(SPARQL)/absences.sparql $@
+	$ROBOT) convert -i $< -o $(BUILD_DIR)/phenoscape-data-kb.ttl \
```

Comment:
This is okay for testing but I think you will want to perform this conversion separately, since you can reuse this for the presences query, and maybe for loading. Maybe just save this as ttl in its existing target?